### PR TITLE
Python fixes

### PR DESCRIPTION
**Pull request by txomon**
For some reason there are a lot of python -c calls everywhere, and there is no a proposed version to use, so I have just made sure that each script runs with the version it needs

### CODE REVIEW

**Comment by txomon** [comment]
I signed it!

**Comment by txomon** [comment]
Fixing same problems as in #19522

I would suggest either checking that 'python -c' calls don't exist anywhere or making travis run on different python default versions

File: cluster/gce/trusty/configure.sh [hunk 1]
```diff
@@ -60,7 +60,7 @@ download_kube_env() {
     -o "${tmp_install_dir}/kube_env.yaml" \
     http://metadata.google.internal/computeMetadata/v1/instance/attributes/kube-env
   # Convert the yaml format file into a shell-style file.
-  eval $(python -c '''
+  eval $(python3 -c '''
 import pipes,sys,yaml
 for k,v in yaml.load(sys.stdin).iteritems():
```
Comment (txomon) [inline cluster/gce/trusty/configure.sh]: I didn't read carefully, this is assuming python2

**Comment by zmerlynn** [comment]
@k8s-bot: ok to test

**Comment by zmerlynn** [comment]
@txomon: Why did you assume any of these are python3? That's unreasonable on 99% of base images.

**Comment by zmerlynn** [comment]
@txomon: We don't have `python3` on the `container-vm` image, so this PR failed the PR builder. You can change everything to `python2`, which is the only Python anyone still cares about. ;)

**Comment by zmerlynn** [comment]
I'm closing this until there's a response to https://github.com/kubernetes/kubernetes/pull/22144#issuecomment-199409110. Thanks for the PR!

**Pull request closed without merging**